src/main/webapp/javascript/login.js

```diff
@@ -0,0 +1,116 @@
+function switchRegistration() {
+    hideError();
+    var checkbox = document.getElementById("register-cb");
+    var checkbox_carRegButton = document.getElementById("register_car-grp");
+    var checkbox_carReg = document.getElementById("register_car-cb");
+    var pwd2 = document.getElementById("password2-grp");
+    var loginBtn = document.getElementById("login-btn");
+    var registerBtn = document.getElementById("register-btn");
+    var name = document.getElementById("name-grp");
+    var surname = document.getElementById("surname-grp");
+    var phoneNumber = document.getElementById("phoneNumber-grp");
+    if (checkbox.checked) {
+        pwd2.classList.remove("w3-hide");
+        name.classList.remove("w3-hide");
+        surname.classList.remove("w3-hide");
+        phoneNumber.classList.remove("w3-hide");
+        loginBtn.classList.add("w3-hide");
+        registerBtn.classList.remove("w3-hide");
+    } else {
+        pwd2.classList.add("w3-hide");
+        loginBtn.classList.remove("w3-hide");
+        registerBtn.classList.add("w3-hide");
+        name.classList.add("w3-hide");
+        surname.classList.add("w3-hide");
+        phoneNumber.classList.add("w3-hide");
```

Comment:
It's a lot of elements to set visibility for. Can some of them be wrapped with another (parent) element, such as `<div>`, so that only the parent element needs to be hidden or shown? E.g. buttons can be separate, but "Name", "Surname", "Phone number" look like they belong in one section.

Reply:
Also - additional functions can be implemented to shorten the code and make it more high-level - e.g.:
```
function hideElement(element) {
    element.classList.add("w3-hide");
}

function showElement(element) {
    element.classList.remove("w3-hide");
}
```